packages/dialog/theme/lumo/vaadin-dialog.js

```diff
@@ -1,2 +1,2 @@
-import './vaadin-dialog-styles.js';
+import './vaadin-dialog-overlay-styles.js';
```

Comment:
This should be reverted.

```suggestion
import './vaadin-dialog-styles.js';
```